vm/src/main/scala/pravda/vm/standard/ExponentialFunction.scala

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (C) 2018  Expload.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package pravda.vm.standard
+
+import pravda.vm.Data.Primitive.{BigInt, Int16, Int32, Int8, Number, Uint16, Uint32, Uint8}
+import pravda.vm.Data.Type
+import pravda.vm.VmError.WrongType
+import pravda.vm.WattCounter.CpuArithmetic
+import pravda.vm._
+
+object ExponentialFunction extends FunctionDefinition {
+  override def id: Long = 0x03L
+
+  override def description: String = "Calculate exponential operation first two items in the stack and push result into"
+
+  override def args: Seq[(String, Seq[Type])] = Seq(
+    "x" -> Seq(Data.Type.Int32),
+    "y" -> Seq(Data.Type.Int32)
+  )
+
+  override val returns = Seq(Data.Type.Number)
+  private val div: Double = math.log10(2.0)
```

Comment:
Very bad naming here. Probably the best solution here just to take `5` as the factor.